Program.cs

```diff
@@ -4,7 +4,10 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hangman Game");
+            string a = "Adam Asmaca";
```

Comment:
a gibi anlamsız kelimeler yerine title gibi anlamlı isimlendirmler yapabiliriz. Rica etsem değiştirebilir misin?

Reply:
Ek olarak şöyle bi bilgi vereyim. Birisi bize düzeltme yapmamız için yorum bırakırsa düzeltmeyi yapıp sonra düzelti gibi yorum bırakıyoruz buranın altına. etiket atarsak daha iyi olur. Uzun lafın kısası düeltmeyi yaptıktan sonra bu kısımdan düzeltildi diye yorum bırakman lazım

Reply:
düzeltildi. @FutureBaseEditor